cypress/integration/tip.test.ts

```diff
@@ -0,0 +1,100 @@
+import { Currency, CurrencyAmount, JSBI, Token, TokenAmount, Trade, WETH } from '@uniswap/sdk'
+import { getFeeAmount } from '../../src/custom/utils/fee'
+
+const stringToCurrency = (amount: string, currency: Currency) =>
+  currency instanceof Token ? new TokenAmount(currency, JSBI.BigInt(amount)) : CurrencyAmount.ether(JSBI.BigInt(amount))
+
+// Assumptions:
+// MAINNET
+// SELL: ETH
+// BUY: DAI
+const INPUT_TOKEN = WETH[1].address
+const OUTPUT_TOKEN = '0x1f9840a85d5af5bf1d1762f925bdaddc4201f984'
+const DAI = '0xc7AD46e0b8a400Bb3C915120d284AafbA8fc4735'
```

Comment:
Confused: if you are buying DAI, why is the OUTPUT_TOKEN !== DAI?

Reply:
fixed